.eslintrc.js

```diff
@@ -8,6 +8,9 @@ module.exports = {
     jest: true,
   },
   settings: {
+    ['import/resolver']: {
```

Comment:
I don't think this needs brackets around it - ['import/resolver']

Reply:
Done